back/polkassembly-front-server/kubernetes/server-namespace.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Namespace  
```

Comment:
don't think this needs to be duplicated here since it's the same namespace.

Reply:
Not sure I get that. do you mean the same namespace as the `server`?

Reply:
you already have a namespace `polkassembly` created from the Polkassembly repo, so why duplicate it here? ... unless you were trying to create another namespace for the server in which case the below lines should be edited.

Reply:
Understood thanks, I've no clue what I'm doing when touching kubernetes config files.